internal/service/alarm_definition_handler_test.go

```diff
@@ -78,6 +78,9 @@ var _ = Describe("Alarm definition handler", func() {
 					Expect(item).To(HaveKey("managementInterfaceId"))
 					Expect(item).To(HaveKey("pkNotificationField"))
 					Expect(item).To(HaveKey("alarmAdditionalFields"))
+					Expect(item).To(HaveKey("alarmLastChange"))
```

Comment:
@pixelsoccupied not sure if they also need to be added to the test handler.

Reply:
@alosadagrande you can consider all the current code Alarm server as deprecated. so feel free to make the minimal changes you need for your tests to pass (if the tests dont need to be updated to pass CI feel free to revert it)

Reply:
ok, i'll leave it like that since it passes my tests :)